### ISSUE TITLE
Is there a way to request write permission of specific files only?

### ISSUE THREAD

**Issue by Muzammil-Bit** [ivehement/saf#30]
I researched the documentation but couldn't find a way to just request permission for 1 or 2 or more files instead of the whole directory. Is there a way if not maybe there should be.

**Comment by jvoltci** [comment]
Yes. Currently, it does not support that. Please feel free to raise your request for your contribution if you'd like.

**Comment by jvoltci** [comment]
2.0.0 adds file pickers: `pickFile()` / `pickFiles(mimeTypes: [...])` return the granted file(s) directly — no whole-directory grant required.
